curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/6555d8faed60b9d3e4a6cefb.md

```diff
@@ -9,7 +9,9 @@ dashedName: step-66
 
 Use `const` and arrow syntax to define a function called `setPlayButtonAccessibleText`. 
 
-This function will set the `aria-label` attribute to the current song, or to the first song in the playlist. And if the playlist is empty, it sets the `aria-label` to `"Play"`.
```

Comment:
In step 68, the description says that `setAttribute()` method should be used on ```playButton``` element, but it's not clear to me what paraphrased version of the deleted paragraph from step 66 would look like.  
I'd like to get some input on that. Thanks

Reply:
@huyenltnguyen  i think, a small definition and an example with very simple wording anywhere around step 66 or 68 should work, this is to let the users know what it is and how it looks like. @jdwilkin4  what do you think ?

Reply:
@JoyShaheb 

It looks like setAttribute was introduced earlier in the project.
So I don't think we need a definition and example here.
If we wanted to add one back when it is first introduced(if it is not there already), then that could be an option. but that is outside the scope of this PR and should be addressed in separate PR

Reply:
@Re1nGer 

This could work for an updated description for step 68. It provides context for why these changes are being made followed by what to do

```md
The `setPlayButtonAccessibleText` function will set the `aria-label` attribute to the current song, or to the first song in the playlist. And if the playlist is empty, it sets the `aria-label` to `"Play"`.

Use the `setAttribute` method on the `playButton` element to set an attribute named `"aria-label"`. For the value, use a ternary to `set song?.title` to `Play ${song.title}` or `"Play"` if there's no `song.title` available.
```